content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,284 @@ weight: 50
 is not required to have access to the cluster IP addresses assigned by Kubernetes.
 
-* The Istio control plane services (Pilot, Mixer, Citadel) and Kubernetes DNS server must be accessible
-from the VMs. This is typically done using an [Internal Load
-Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).
-You can also use NodePort, run Istio components on VMs, or use custom network configurations,
-separate documents will cover these advanced configurations.
+* Mesh expansion VMs must have access to a DNS server that resolves names to cluster IP addresses. Options
+include exposing the Kubernetes DNS server through an internal load balancer, using a Core DNS
+server, or configuring the IPs in any other DNS server accessible from the VM.
+
+* If you haven't already enabled mesh expansion [at install time with Helm](/docs/setup/kubernetes/helm-install/), you have installed the [Helm client](https://docs.helm.sh/using_helm/). You'll need it to enable mesh expansion for the cluster.
 
 ## Installation steps
 
 Setup consists of preparing the mesh for expansion and installing and configuring each VM.
 
-An example script to help with Kubernetes setup is available as part of the release bundle called
-[`install/tools/setupMeshEx.sh`]({{< github_file >}}/install/tools/setupMeshEx.sh). Check the script content and environment variables supported (like GCP_OPTS).
-
-An example script to help configure a machine is available as part of the release bundle called [`install/tools/setupIstioVM.sh`]({{< github_file >}}/install/tools/setupIstioVM.sh).
-You should customize it based on your provisioning tools and DNS requirements.
-
 ### Preparing the Kubernetes cluster for expansion
 
-*   Setup Internal Load Balancers (ILBs) for Kube DNS, Pilot, Mixer and Citadel. This step is specific to
-each cloud provider, so you may need to edit annotations. You can use an ILB based on Keepalived at
-[here](https://github.com/gyliu513/work/tree/master/k8s/charts/keepalived) for demo or test in case where
-the cloud provider or private cloud (for example IBM Cloud Private) doesn't have load balancer service
-support out of box.
+The first step when adding non-Kubernetes services to an Istio mesh is to configure the Istio installation itself and generate the configuration that will allow it to be used by the mesh expansion VMs. To prepare the cluster for mesh expansion, run the following commands on a machine with cluster admin privileges:
+
+1.    Ensure that mesh expansion is enabled for the cluster. If you did not specify `--set global.meshExpansion=true` at
+    install with Helm, you can either `helm upgrade` with the new option (note that this will only work if you originally
+    installed with Helm and Tiller), or you can use `helm template` to update your configuration with the option and reapply
+    with `kubectl`. 
+
+        {{< text bash >}}
```

Comment:
This line should have exactly 8 spaces before it but it has 9. Thus, the indentation breaks and the code block is not rendered.

Reply:
If you want both code blocks to have syntax highlighting, the indentation on lines 39-54 has to be 4 spaces.

Reply:
Fixed.